build_tools/github_actions/build_dist.py

```diff
@@ -0,0 +1,290 @@
+      f"-DCMAKE_INSTALL_PREFIX={INSTALL_DIR}",
+      f"-DCMAKE_BUILD_TYPE=Release",
+      # TODO: Should be able to build runtime without compiler if resolved:
+      # https://github.com/google/iree/issues/4617
+      f"-DIREE_BUILD_COMPILER=OFF",
+      f"-DIREE_BUILD_PYTHON_BINDINGS=ON",
+      f"-DIREE_BUILD_SAMPLES=OFF",
+      f"-DIREE_BUILD_TESTS=OFF",
+  ])
+
+  print("*** Building ***")
+  subprocess.check_call([
+      sys.executable,
+      CMAKE_CI_SCRIPT,
+      "--build",
+      BUILD_DIR,
+      "--target",
+      "install-IreePythonPackage-rt-stripped",
+  ])
+
+
+def build_py_xla_compiler_tools_pkg():
+  """Builds the iree-install/python_packages/iree_tools_xla package."""
+  # Clean up install and build trees.
+  shutil.rmtree(INSTALL_DIR, ignore_errors=True)
+  shutil.rmtree(os.path.join(BUILD_DIR, "CMakeCache.txt"), ignore_errors=True)
+
+  # CMake configure.
+  print("*** Configuring ***")
+  subprocess.check_call([
+      sys.executable,
+      CMAKE_CI_SCRIPT,
+      f"-B{BUILD_DIR}",
+      f"-DCMAKE_INSTALL_PREFIX={INSTALL_DIR}",
+      f"-DCMAKE_BUILD_TYPE=Release",
+      "-DIREE_BUILD_XLA_COMPILER=ON",
+      f"-DIREE_BUILD_PYTHON_BINDINGS=ON",
+      f"-DIREE_BUILD_SAMPLES=OFF",
+      f"-DIREE_BUILD_TESTS=OFF",
+  ])
+
+  print("*** Building ***")
+  subprocess.check_call([
+      sys.executable,
+      CMAKE_CI_SCRIPT,
+      "--build",
+      BUILD_DIR,
+      "--target",
+      "install-IreePythonPackage-tools-xla-stripped",
+  ])
+
+
+def build_py_tflite_compiler_tools_pkg():
+  """Builds the iree-install/python_packages/iree_tools_tflite package."""
+  # Clean up install and build trees.
+  shutil.rmtree(INSTALL_DIR, ignore_errors=True)
+  shutil.rmtree(os.path.join(BUILD_DIR, "CMakeCache.txt"), ignore_errors=True)
+
+  # CMake configure.
+  print("*** Configuring ***")
+  subprocess.check_call([
+      sys.executable,
+      CMAKE_CI_SCRIPT,
+      f"-B{BUILD_DIR}",
+      f"-DCMAKE_INSTALL_PREFIX={INSTALL_DIR}",
+      f"-DCMAKE_BUILD_TYPE=Release",
+      "-DIREE_BUILD_TFLITE_COMPILER=ON",
+      f"-DIREE_BUILD_PYTHON_BINDINGS=ON",
+      f"-DIREE_BUILD_SAMPLES=OFF",
+      f"-DIREE_BUILD_TESTS=OFF",
+  ])
+
+  print("*** Building ***")
+  subprocess.check_call([
+      sys.executable,
+      CMAKE_CI_SCRIPT,
+      "--build",
+      BUILD_DIR,
+      "--target",
+      "install-IreePythonPackage-tools-tflite-stripped",
+  ])
+
+
+def build_py_tf_compiler_tools_pkg():
+  """Builds the iree-install/python_packages/iree_tools_tf package."""
+  # Clean up install and build trees.
+  shutil.rmtree(INSTALL_DIR, ignore_errors=True)
+  shutil.rmtree(os.path.join(BUILD_DIR, "CMakeCache.txt"), ignore_errors=True)
+
+  # CMake configure.
+  print("*** Configuring ***")
+  subprocess.check_call([
+      sys.executable,
+      CMAKE_CI_SCRIPT,
+      f"-B{BUILD_DIR}",
+      f"-DCMAKE_INSTALL_PREFIX={INSTALL_DIR}",
+      f"-DCMAKE_BUILD_TYPE=Release",
+      "-DIREE_BUILD_TENSORFLOW_COMPILER=ON",
```

Comment:
no f?

Reply:
copy-pasta. Made them all prefixed for consistency.